doc/lua_api.txt

```diff
@@ -3701,6 +3706,9 @@ Definition tables
 	  when displacement mapping is used
 	  Directions are from the point of view of the tile texture,
 	  not the node it's on
+* `{name="image.png", color="red"}`
```

Comment:
This works with color names, hex colors and tables containing `r`, `g` and `b` keys.